index.html

```diff
@@ -324,20 +328,27 @@ <h2>Contact me</h2>
                                                 required="required"></textarea>
                                         </li>
                                     </ul>
-
+                                    <div class="MyBtn">
+                                        <div class="click53">
+                                            <button class="me" id="submit">Get in touch</button>
+                                        </div>
+                                    </div>
```

Comment:
- [ ] It would be nice to include the attribute `type="submit"` here, that way, you can submit the form after the button has been clicked.

Reply:
Hello @Gambit142, @BenMukebo in regards to the changes you requested. I have 
1.  I have implemented the Validation form
2. Added a p tag just below the button tag 
3. inserted the type=submit attribute